ap_api/resources/get-quote.api.mdx

```diff
@@ -0,0 +1,47 @@
+
+StellarId's are objects that identify end-users and SEP-31 Sending Anchors, but not SEP-31 Receiving Anchors.
+
+For a SEP-12 customer, the `id` field should be sufficient to fully identify the customer in the Anchor Backend.
+
+For a SEP-31 Sending Anchor, the `account` field should be used.
+
+
+</div><SchemaItem collapsible={false} name={"id"} required={false} schemaName={"string"} qualifierMessage={undefined} schema={{"type":"string","description":"The `id` of the customer registered through SEP-12."}}></SchemaItem><SchemaItem collapsible={false} name={"account"} required={false} schemaName={"string"} qualifierMessage={undefined} schema={{"type":"string","description":"Either the Stellar account or Muxed account address of the on-chain entity."}}></SchemaItem></div></details></SchemaItem><SchemaItem collapsible={false} name={"transaction_id"} required={false} schemaName={"string"} qualifierMessage={undefined} schema={{"type":"string"}}></SchemaItem><SchemaItem collapsible={false} name={"created_at"} required={false} schemaName={"date-time"} qualifierMessage={undefined} schema={{"type":"string","format":"date-time"}}></SchemaItem></ul></details></TabItem><TabItem label={"Example (from schema)"} value={"Example (from schema)"}><ResponseSamples responseExample={"{\n  \"id\": \"string\",\n  \"sell_amount\": \"string\",\n  \"sell_asset\": \"string\",\n  \"buy_amount\": \"string\",\n  \"buy_asset\": \"string\",\n  \"expires_at\": \"2023-03-23\",\n  \"price\": \"string\",\n  \"total_price\": \"string\",\n  \"creator\": {\n    \"id\": \"string\",\n    \"account\": \"string\"\n  },\n  \"transaction_id\": \"string\",\n  \"created_at\": \"2023-03-23\"\n}"} language={"json"}></ResponseSamples></TabItem></SchemaTabs></TabItem></MimeTabs></div></TabItem><TabItem label={"404"} value={"404"}><div>
```

Comment:
We should be able to develop a jsx component that reads from a file path right? Did you autogenerate this content?

Reply:
This is how it works 
https://github.com/stellar/stellar-docs/pull/94/files#diff-1bf4a754c9286275407afcadab70535f0cff763e30111ecdeb733d6b6770f785